.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @luke-zhu @arthur-observe @stefandorsett
+* @terraform-maintainers
```

Comment:
Invalid, `@observeinc/terraform-maintainers` would be the valid representation of this. 

That aside, the intended purpose of Terraform maintainers is for Terraform expertise. Definitely the provider,  some reusable GHA workflows for Terraform, but not every single Terraform module. This is a broad issue at Observe (anything even tangentially related to Terraform is likely to be reported to the integrations team) so I'm pretty sensitive to not reinforcing those bad heuristics.

I've clarified that in the team description:

https://github.com/orgs/observeinc/teams/terraform-maintainers

Reply:
:thinking: do we have an integrations team that would be better suited as codeowners? I'm impartial to doing terraform + integration?

Reply:
There is no suitable team, I would recommend making one. There's a tragedy of the commons problem here. If you take an expansive view of code ownership (put Terraform + integrations), it either:

* Imposes a large amount of noise, if you're using teams without automated review assignment
* Decreases the odds that you get a qualified reviewer, because the pool is too broad

You want enough people to balance the workload and ensure that a reviewer is available but not so many that you are imposing excess cost (noise). 

1 is never enough in a corporate environment, beyond the tiniest startup. 5 is probably a sweet spot if you have that many qualified people. 10 is a rough upper limit. The [two pizza team](https://docs.aws.amazon.com/whitepapers/latest/introduction-devops-aws/two-pizza-teams.html) rule holds here.

Can't remember off the top of my head how permissions for GH teams works but if you need help creating I can handle.

Happy to take this to the team channel in Slack for commentary/a vote as well. My thought here is that we need a `cloud` team under integrations. That could have `aws`, `google`, and `azure` child teams, but that specificity is probably premature.